.github/workflows/update-fetchgit.yml

```diff
@@ -0,0 +1,95 @@
+name: Update fetchGit repositories
+
+on:
+  push:
+
+  pull_request:
+
+  workflow_dispatch:
+
+  schedule:
+    # run three times a day
+    - cron: "0 4,12,20 * * *"
+
+jobs:
+  update-nix-fetchgit:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # for git push
+      contents: write
+      # to allow opening the PR
+      pull-requests: write
+
+    env:
+      BRANCH_NAME: "update-fetchgit"
+
+    steps:
+      - name: Install Nix
+        uses: cachix/install-nix-action@v25
+        with:
+          # needed at least for act
+          # TODO: I have no idea how to properly fill this variable... sigh...
+          #nix_path: nixpkgs=channel:nixpkgs-unstable
```

Comment:
In this case, using nixpkgs-unstable (instead of nixos-unstable) is perfectly fine, as we don't want to build OS updates here.

Reply:
Please comment with the required change. I tried to set it as shown, but it just didn't yield the intended result. I learned that it's necessary to use `nixpkgs*` instead of `nixos*` anyway on non-Nix systems.

Reply:
I'll test later, I was under the assumption that this was all that's needed for the action to properly configure the channels.

Reply:
Took a bit longer to have the time to review this.

As for the `nix_path` thing: Turns out this is broken since September 2020 (and noone really noticed, since channels aren't really the way to go, anymore (you'd rather use flakes and specifying inputs, but that's another story)).

As such, I'd suggest to remove the `nix_path` definition entirely, to avoid confusion.